code/datums/components/mood.dm

```diff
@@ -235,6 +243,9 @@
 			master.remove_movespeed_modifier(MOVESPEED_ID_SANITY)
 			master.add_actionspeed_modifier(/datum/actionspeed_modifier/high_sanity)
 			sanity_level = 1
+
+	//around 0.5 per tick
+	setMentalEntropy(mental_entropy + (SANITY_MAXIMUM/2 - sanity)/300)
```

Comment:
Don't do math per tick, do it per seconds using deltatime.

Reply:
Also I'm against a third var besides sanity. Use sanity for this, Sanity is mood over a long time, it makes no sense to add another sense to make mood even more confusing to both players and coders.

Reply:
i mean i guess i could use sanity, but i would have to make it harder to get 0 sanity, currently all you have to do to reach 0 sanity is be hungry for 5 or so minutes. I dont know what is the better way. i cant really do what this is supposed to do without a 3rd variable or a major rework of how mood fundamentally works

Reply:
What you can do which I'd be okay with because it's not another variable is add a timer that starts ticking below a certain threshold, if your sanity goes back up it deletes but if it runs out you get a disorder.

Reply:
I'd say make it a chance that increases while you remain in low sanity.